### PR TITLE
minor edits to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by steveburnett**
## Description
Minor edits to CONTRIBUTING.md: 
* the link in "Make sure your code follows the [code style guidelines](https://github.com/prestodb/presto#code-style)," was broken and the correct destination was https://github.com/prestodb/presto/blob/master/CONTRIBUTING.md#code-style . 
* Also updated the `beginner-task` tag recommendation to `good-first-issue`.

## Motivation and Context
Noticed that the link in "Make sure your code follows the [code style guidelines](https://github.com/prestodb/presto#code-style)," was broken. 

(Also noticed the same link was broken in [Presto Development Guidelines](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#development) and corrected that link by editing the wiki.)

## Impact
Readers of CONTRIBUTING.md can click the link and view the [code style guidelines](https://github.com/prestodb/presto#code-style), as intended. 

## Test Plan
Testing was a careful review of the edited file before pushing the branch, and again after creating the PR. 

## Contributor checklist

- [x] Please make sure your submission complies with our [development](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#development), [formatting](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#formatting), [commit message](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#commit-formatting-and-pull-requests), and [attribution guidelines](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#attribution).
- [x] PR description addresses the issue accurately and concisely.  If the change is non-trivial, a GitHub Issue is referenced.
- [x] Documented new properties (with its default value), SQL syntax, functions, or other functionality.
- [x] If release notes are required, they follow the [release notes guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines).
- [x] Adequate tests were added if applicable.
- [x] CI passed.

## Release Notes
Please follow [release notes guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines) and fill in the release notes below.

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

**Comment by steveburnett** [comment]
Initially [linux-build-and-unit-test failed](https://app.circleci.com/pipelines/github/prestodb/presto/11885/workflows/a5ae83b5-2990-4809-bb83-b02e0f9a61b1/jobs/45300) and re-run wasn't a configured option. 

Assuming this was another intermittent test failure, I made another small change then committed, squashed merge commit, and pushed to re-start the test run. 

[The re-run passed](https://app.circleci.com/pipelines/github/prestodb/presto/11904/workflows/6f2dcb55-2645-4400-8780-dbe6fd3b4c1b/jobs/45402).

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Thanks for your interest in Presto.  Our goal is to build a fast, scalable and r
 
 ## Getting Started
 
-Presto's [open issues are here](https://github.com/prestodb/presto/issues). Issues that would make a good first pull request for new contributors with the `beginner-task` tag. An easy way to get started helping the project is to *file an issue*. Issues can include bugs, new features, or documentation that looks outdated. For community support, [ask for help in Slack](https://join.slack.com/t/prestodb/shared_invite/enQtNTQ3NjU2MTYyNDA2LTYyOTg3MzUyMWE1YTI3Njc5YjgxZjNiYTgxODAzYjI5YWMwYWE0MTZjYWFhNGMwNjczYjI3N2JhM2ExMGJlMWM).
+Presto's [open issues are here](https://github.com/prestodb/presto/issues). Issues that would make a good first pull request for new contributors with the `good-first-issue` tag. An easy way to get started helping the project is to *file an issue*. Issues can include bugs, new features, or documentation that looks outdated. For community support, [ask for help in Slack](https://join.slack.com/t/prestodb/shared_invite/enQtNTQ3NjU2MTYyNDA2LTYyOTg3MzUyMWE1YTI3Njc5YjgxZjNiYTgxODAzYjI5YWMwYWE0MTZjYWFhNGMwNjczYjI3N2JhM2ExMGJlMWM).
```
Comment (tdcmeehan) [inline CONTRIBUTING.md:14]: Maybe link to the issue: https://github.com/prestodb/presto/labels/good%20first%20issue

Also there are no dashes

**Comment by tdcmeehan** [comment]
LGTM % nit!

**Comment by steveburnett** [comment]
[linux-presto-e2e-tests](https://app.circleci.com/pipelines/github/prestodb/presto/11940/workflows/a328b486-d562-44d5-b17a-32d947696982/jobs/45573) and [linux-spark-e2e-tests](https://app.circleci.com/pipelines/github/prestodb/presto/11940/workflows/a328b486-d562-44d5-b17a-32d947696982/jobs/45572), which passed previously, failed this time through. To get past this intermittent failure I made a minor change to the file, committed, squashed commits, and force-pushed to re-run the tests.

File: CONTRIBUTING.md (hunk 1 above)
Comment (steveburnett) [reply CONTRIBUTING.md]: > Maybe link to the issue: [ good first issue](https://github.com/prestodb/presto/labels/good%20first%20issue)
> 
> Also there are no dashes

I didn't think of linking to the tag! Done.

**Comment by steveburnett** [comment]
> [linux-presto-e2e-tests](https://app.circleci.com/pipelines/github/prestodb/presto/11940/workflows/a328b486-d562-44d5-b17a-32d947696982/jobs/45573) and [linux-spark-e2e-tests](https://app.circleci.com/pipelines/github/prestodb/presto/11940/workflows/a328b486-d562-44d5-b17a-32d947696982/jobs/45572), which passed previously, failed this time through. To get past this intermittent failure I made a minor change to the file, committed, squashed commits, and force-pushed to re-run the tests.

Worked! I don't know why the tests fail intermittently but they all passed this time.

**Review by tdcmeehan — APPROVED** [review]

**Pull request merged**